cmd/flags.go

```diff
@@ -77,12 +77,10 @@ func initGlobalConfigFlags(cmd *cobra.Command, configFilePath *string) {
 // start command to run the client against test networks. Only one flag
 // from this set is allowed.
 func initGlobalNetworkFlags(cmd *cobra.Command) {
-	// TODO: Remove the mainnet flag.
-	// DEPRECATED
 	cmd.PersistentFlags().Bool(
 		network.Mainnet.String(),
 		false,
-		"Mainnet network (DEPRECATED)",
+		"Mainnet network",
```

Comment:
On line 99 we have a comment about removing the mainnet flag. Can you pls also remove it?

Reply:
See https://github.com/keep-network/keep-core/pull/3690/commits/ca8bdb802d7f2e04ddafa1047de60d28b6776dea